### PR TITLE
Animations 2

### LINKED ISSUE #112: Fix multiple spring configurations in a view
This will currently default to only using the spring from color at the end in Text:
```
Vertical
backgroundColor red
opacity 0.75
translateX 10
translateY 0
rotateX 10
onClick <
when <isOn
opacity 1 spring bounciness 10 speed 1
translateX 20 spring bounciness 15 speed 2
translateY 30
rotateX 20 easeIn
Text
color red
text hey
when <isOn
color blue spring
```

This will take the last spring:
```
Vertical
backgroundColor red
opacity 0.75
translateX 10
translateY 0
rotateX 10
onClick <
when <isOn
opacity 1 spring bounciness 10 speed 1
translateX 20 spring bounciness 15 speed 2
translateY 30
rotateX 20 easeIn
Text
text hey
```

I have a fix for the second case already. Need to work out a fix for the first one too.

### PR DESCRIPTION
**Pull request by dariocravero**
This PR fixes #112, #98 and #108. It also supports `Timing` animations in native and fixes `delay` on DOM since `react-spring` supports it already. It also fixes `easeInOut`.

To wrap up animations, I'd say we're missing the following:
- support spring on `hover` #107,
- add `onAnimationDone` callback #111,
- add `before` and `after` system scopes - this should allow for in/out transitions, and
- stagger cards in lists.

The last two can be done with a logic layer in JS so I wouldn't say there's a lot of rush on them.

Am I missing anything @amymc, @tombrewsviews?

NOTE: After releasing this version, we'd need to update `@viewstools/use` to add `@viewstools/animations` as a dependency.

### CODE REVIEW

File: morph/react/get-dependencies.js [hunk 1]
```diff
@@ -29,7 +29,8 @@ export default (state, getImport) => {
     }
   }
 
-  const dependencies = []
+  const dependencies = [`import React from 'react'`]
```
Comment (amymc) [inline morph/react/get-dependencies.js]: do we need this on native now as well?

File: morph/react/get-body.js [hunk 2]
```diff
@@ -20,81 +22,83 @@ export default ({ state, name }) => {
...
-              )
+        if (!state.isReactNative && curve !== 'spring') return
+
+        const tag = curve === 'spring' ? 'Spring' : 'Timing'
+        let config = Object.entries(configValues)
+          .map(([key, value]) => `${key}={${value}}`)
+          .join(' ')
+
+        if (curve !== 'spring') {
+          config = `curve="${curve}" ${config}`
```
Comment (amymc) [inline morph/react/get-body.js]: When is this used? I know it's for timing animations, but it's not appearing in the compiled output when I try timing animations.

File: morph/react/get-dependencies.js (hunk 1 above)
Comment (dariocravero) [reply morph/react/get-dependencies.js]: It was always there, I just moved it in the deps now so we can sort them properly :). RN is a target like DOM, they share the react package though

File: morph/react/get-body.js (hunk 2 above)
Comment (dariocravero) [reply morph/react/get-body.js]: For native. See the check on line 35 that bails if it isn’t RN.

**Pull request merged**